src/main/java/org/owasp/validator/css/CssScanner.java

```diff
@@ -96,6 +96,18 @@
 
     private static final Pattern p = Pattern.compile(CDATA, Pattern.DOTALL);
 
+    /**
+     * Constructs a scanner based on the given policy.
```

Comment:
Can you add a comment to the constructor description that indicates this version of the constructor defaults shouldParseImportedStyles to false (the default) and they should look at the next constructor for a description of what that feature does.

Reply:
Done

Reply:
Should we mark this constructor as deprecated to delete it in the future?